lib/src/chart/line_chart/line_chart_painter.dart

```diff
@@ -812,7 +814,7 @@ class LineChartPainter extends AxisChartPainter<LineChartData>
 
         final double bottomChartPadding = getExtraNeededVerticalSpace() - getTopOffsetDrawSize();
         final Offset to =
-        Offset(getPixelX(line.x, chartUsableSize), viewSize.height - bottomChartPadding);
+            Offset(getPixelX(line.x, chartUsableSize), viewSize.height - bottomChartPadding);
```

Comment:
Useless change.

Reply:
this is from a bug in dart formatter where it had previously not formatted this line. the only way i can prevent this from occurring is to turn off dart formatter.